qhub/template/stages/03-kubernetes-initialize/modules/traefik_crds/main.tf

```diff
@@ -548,6 +548,11 @@ resource "kubernetes_manifest" "middleware" {
                       type = "object"
                       properties = {
                         customRequestHeaders = {
+                          properties = {
```

Comment:
Is there a reason this isn't

```
additionalProperties:
      type: string
```

Like https://github.com/traefik/traefik-helm-chart/blob/master/traefik/crds/middlewares.yaml#L468

Reply:
The second one should be like this too